app/assets/styles/app.scss

```diff
@@ -3,11 +3,61 @@ $mobile-breakpoint:        768px;
 $tablet-breakpoint:        992px;
 $small-monitor-breakpoint: 1200px;
 
+$teal: #56A5BF;
+$dark-teal: #1C7592;
+$light-teal: #7CC1D8;
+$dark-gray: #222222;
+$white: #FFFFFF;
+$red: #FF684C;
+
 // General nav
 nav {
     margin-bottom: 40px !important;
 }
 
+#CAC_logo {
+	content: url("../../static/images/CAC_logo.png");
+	height: 60px;
+	width: auto;
+	margin-right: 15px;
+	margin-left: 10px;
+}
+
+.ui.secondary.pointing.menu {
+	background-color: $teal;
+	width: 100%;
+	font-size: 1.3em;
+}
+
+.ui.secondary.pointing.menu > .header.item {
+	color: $white !important;
+	font-size: 1.4em;
+	font-weight: 400;
+	border: none;
+}
+
+.ui.secondary.pointing.menu > .right.menu {
+	margin-right: 20px;
```

Comment:
General question for a bunch of this sass code: Is hardcoding pixel values okay? Will it look right on all screens?